include/mbedtls/ssl_internal.h

```diff
@@ -256,6 +256,22 @@
       + ( MBEDTLS_SSL_CID_OUT_LEN_MAX ) )
 #endif
 
+#if defined(MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH)
+static inline uint32_t mbedtls_ssl_get_output_buflen( const mbedtls_ssl_context *ctx )
+{
+    return (uint32_t) mbedtls_ssl_get_max_frag_len( ctx )
+               + MBEDTLS_SSL_HEADER_LEN + MBEDTLS_SSL_PAYLOAD_OVERHEAD
+               + MBEDTLS_SSL_CID_OUT_LEN_MAX;
```

Comment:
In commit "Add static functions to calculate ssl buffer sizes", we probably don't want to add MBEDTLS_SSL_CID_OUT_LEN_MAX unless CID is configured. Or, we should define MBEDTLS_SSL_CID_OUT_LEN_MAX to 0 when CID is not configured and avoid adding any ifdefs here or later.

Reply:
I'll move my fix to this commit.